core/nullDC.cpp

```diff
@@ -39,35 +39,38 @@ settings_t settings;
 #include <windows.h>
 #endif
 
-int GetFile(char *szFileName, char *szParse=0,u32 flags=0) 
+int GetFile(std::string& diskFileName)
 {
-	cfgLoadStr("config","image",szFileName,"null");
-	if (strcmp(szFileName,"null")==0)
```

Comment:
@LoungeKatt the null check is now inside thr #if, but there was no handling other than the Windows one previously, so no functionality removed here.

Reply:
@baka0815 Correct, but https://github.com/reicast/reicast-emulator/blob/055c3599b6cfa99279542dc2d29c6e5032995ded/core/hw/naomi/naomi_cart.cpp#L209 should also have been changed to match.

Reply:
I'd leave the #ifdef inside the if - only the file open logic is platform specific, not the empty string chech

Reply:
@skmp there would be a nullpointer check which would do nothing then (everything in it would be inside the ``#ifdef``, so what's the point keeping it?